File: tests/kotlin/kotlin_while_results/test_while_break.kt
// vybe-test: kotlin/kotlin_while_results/test_while_break
// origin: languages/kotlin/tests/kotlin/test_kotlin_while_results.rs

fun main() {
            var i = 0
            while (true) {
                if (i == 2) {
                    break
                }
                i = i + 1
            }
            println(i)
        }

File: tests/lua/loops_while/test_while_break.lua
-- vybe-test: lua/loops_while/test_while_break
-- origin: languages/lua/tests/lua/test_loops_while.rs

local __w1 = "123"
local __i = 0

local i=1; local s=''; while true do s=s..i; if i==3 then break end; i=i+1 end; do local __t = tostring(s); __i = __i + 1
  if __i == 1 and __t ~= __w1 then error("FAIL: want [" .. __w1 .. "] got [" .. __t .. "]") end end

if __i == 0 then error("FAIL: no output, wanted [" .. __w1 .. "]") end
